modules/swagger-codegen/src/main/resources/dart/pubspec.mustache

```diff
@@ -4,7 +4,7 @@ description: {{pubDescription}}
 dependencies:
   http: '>=0.11.1 <0.12.0'
   dartson: "^0.2.4"
-  intl: "^0.12.4+2"
+  intl: any
```

Comment:
That would work just as well, so do as you think!

However, the choice of that version seems quite arbitrary. I am quite sure the features that are used have been present in much older versions of the library as well.